VideoIndexerEnabledByArc/aks/readme.md

```diff
@@ -146,26 +146,31 @@ You will recieve a response of the following format:
     "translatorCognitiveServicesSecondaryKey": "<key>",
```

Comment:
can you place the speeches together , the tranlsator together and the ocr's together ?